borg/archiver.py

```
@@ -1488,6 +1488,10 @@ def sig_info_handler(signum, stack):  # pragma: no cover
             break
 
 
+def sig_term_handler(signum, stack):
+    raise KeyboardInterrupt()
```

dirty. maybe raise a separate exception?